apps/collabcal/src/redux/slices/account.ts

```diff
@@ -45,26 +44,31 @@ export const accountSlice = createSlice({
   } as AccountState,
   reducers: {},
   extraReducers(builder) {
-    builder
-      .addCase(loginWithEmail.fulfilled, (state, action) => ({
-        ...state,
-        status: "idle",
-        session: action.payload,
-      }))
-      .addCase(fetchSession.fulfilled, (state, action) => ({
-        ...state,
-        status: "idle",
-        session: action.payload,
-      }))
-      .addCase(logout.fulfilled, (state) => ({
-        ...state,
-        status: "idle",
-        session: null,
-      }));
+    builder.addCase(fetchSession.fulfilled, (state, action) => {
+      state.session = action.payload;
+      state.status = "idle";
+    });
+    builder.addCase(logout.fulfilled, (state) => ({
+      ...state,
+      status: "idle",
+      session: null,
+    }));
 
     builder
+      .addMatcher(
+        (action) => /^account\/login\/\w+\/fulfilled$/.test(action.type),
```

Comment:
Anything like `account/login/google/fulfilled` for example will pass this regex.